taprpc/taprootassets.proto

```diff
@@ -611,6 +611,23 @@ message ListTransfersResponse {
     repeated AssetTransfer transfers = 1;
 }
 
+// BlockHash represents a hash of a Bitcoin (BTC) blockchain block.
+//
+// This message includes two separate fields for the same hash:
+// - The `hash` field provides the raw byte format, which is convenient for Go
+//   developers to use directly in other RPC calls.
+// - The `reversed_hash_hex` field provides the byte-reversed hash in
+//   hexadecimal string format, which is useful for users who need to copy and
+//   paste the value, such as passing it as a command-line argument.
+message BlockHash {
```

Comment:
If we do this, it should actually be called `ChainHash`, as this representation also applies to transaction hashes.

I think this file is the correct place for it, so it can be included in all sub RPC packages.